WordPress/src/main/java/org/wordpress/android/ui/reader/repository/usecases/GetDiscoverCardsUseCase.kt

```diff
@@ -3,33 +3,38 @@ package org.wordpress.android.ui.reader.repository.usecases
 import dagger.Reusable
 import kotlinx.coroutines.CoroutineDispatcher
 import kotlinx.coroutines.withContext
-import org.wordpress.android.datasets.ReaderDiscoverCardsTable
-import org.wordpress.android.datasets.ReaderPostTable
+import org.wordpress.android.datasets.ReaderDiscoverCardsTableWrapper
+import org.wordpress.android.datasets.wrappers.ReaderPostTableWrapper
+import org.wordpress.android.fluxc.utils.AppLogWrapper
 import org.wordpress.android.models.discover.ReaderDiscoverCard
 import org.wordpress.android.models.discover.ReaderDiscoverCard.InterestsYouMayLikeCard
 import org.wordpress.android.models.discover.ReaderDiscoverCard.ReaderPostCard
 import org.wordpress.android.models.discover.ReaderDiscoverCards
 import org.wordpress.android.modules.IO_THREAD
 import org.wordpress.android.ui.reader.ReaderConstants
+import org.wordpress.android.util.AppLog.T.READER
 import javax.inject.Inject
 import javax.inject.Named
 
 @Reusable
 class GetDiscoverCardsUseCase @Inject constructor(
     private val parseDiscoverCardsJsonUseCase: ParseDiscoverCardsJsonUseCase,
+    private val readerDiscoverCardsTableWrapper: ReaderDiscoverCardsTableWrapper,
+    private val readerPostTableWrapper: ReaderPostTableWrapper,
+    private val appLogWrapper: AppLogWrapper,
     @Named(IO_THREAD) private val ioDispatcher: CoroutineDispatcher
 ) {
     suspend fun get(): ReaderDiscoverCards =
             withContext(ioDispatcher) {
-                val cardJsonList = ReaderDiscoverCardsTable.loadDiscoverCardsJsons()
+                val cardJsonList = readerDiscoverCardsTableWrapper.loadDiscoverCardsJsons()
                 val cards: ArrayList<ReaderDiscoverCard> = arrayListOf()
 
                 if (cardJsonList.isNotEmpty()) {
```

Comment:
Quick question, I normally test for logic like what happens if the `cardJsonList` is empty but in this case, nothing happens so since it's obvious is it not worth testing?

Reply:
Great catch, definitely worth adding a test just to be sure - 66f7a9314c5d6d620387ca60ff34466f69fb0f42